cds_ils/importer/importer.py

```diff
@@ -243,26 +244,22 @@ def import_summary(self):
             "series": self.series_list,
             "raw_json": self.json_data,
             "document_json": doc_json,
-            "document": self.document
+            "document": self.document,
         }
 
     def preview_delete(self):
         """Preview deleting a record."""
         self._validate_provider()
-        self.action = "update"
+        self.action = "none"
         # finds the exact match, update records
         self.document = self._match_document()
 
         if self.document:
             self.output_pid = self.document["pid"]
             self.eitem_summary = \
                 self.eitem_importer.preview_delete(self.document)
-            document_has_only_serial_relations = \
```

Comment:
this check can be removed only if we raise an exception for "multipart_monograph" relation reference, which I think it is not added below